File: converter/src/main/java/com/example/converter/presentataion/controller/TestController.java
package com.example.converter.presentataion.controller;

import com.example.converter.presentataion.dto.DataRequest;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class TestController {

    @GetMapping("/hello")
    public ResponseEntity<DataRequest> request(DataRequest dataRequest) {
        return ResponseEntity.ok(dataRequest);
    }
}

File: retry-test/src/main/kotlin/com/example/retrytest/controller/TestController.kt
package com.example.retrytest.controller

import com.example.retrytest.caller.TestCaller
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestParam
import org.springframework.web.bind.annotation.RestController

@RestController
class TestController(
    private val testCaller: TestCaller
) {

    @GetMapping("/v1/test")
    fun callTest(@RequestParam isTest: Boolean) = ResponseEntity.ok(testCaller.callAll(isTest))
}